app.js

```diff
@@ -0,0 +1,92 @@
+const billAmount = document.querySelector(".bill-amount");
```

Comment:
I like the way you structured the script code so it can be read from top to bottom nicely.